cli.js

```diff
@@ -1,6 +1,19 @@
 #!/usr/bin/env node
 'use strict';
 
+// Prefer the local installation of AVA.
+var resolveFrom = require('resolve-from');
+var localCLI;
+try {
+	localCLI = resolveFrom('.', 'ava/cli');
+} catch (e) {}
+
+if (localCLI && localCLI !== __filename) {
+	console.warn('Using local install of AVA.');
```

Comment:
I don't think we need that warning, it gives no helpful information to the test output. Plus, it will not look good.